.lighthouserc.js

```diff
@@ -0,0 +1,234 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
```

Comment:
Could this be a `.ts` file?

Reply:
No, but we can change it to JSON if we want

Reply:
Let's do that then.

Reply:
Done